Documentation/Reference.md

```diff
@@ -61,51 +61,117 @@ Where `foo` is an executable and `bar.a` a static library.
 
 ## Package Manifest File Format Reference
 
-Instructions for how to build a package are provided by a manifest file, called `Package.swift`. You can customize this file to declare build targets or dependencies, include or exclude source files, and specify build configurations for the module or individual files.
+Instructions for how to build a package are provided by the `Package.swift` manifest file. `Package.swift` is a Swift file that declaratively configures a Package using types defined in the `PackageDescription` module. It currently has following properties:
 
-Here's an example of a `Package.swift` file:
+### name
+
+Defines the name of the package and is the minimal requirement for a manifest to be valid. When the sources are located directly under `Sources/` directory, there is only one module and the module name is same as the package name.
```

Comment:
I'd move the second sentence to after the example. As a general pattern I think the reference should look like:

<short description>
<example>
<long/verbose description>